packages/mobile/src/screens/edit-playlist-screen/CreatePlaylistScreen.tsx

```diff
@@ -56,14 +57,16 @@ const initialErrors = {
 
 export const CreatePlaylistScreen = () => {
   const { toast } = useToast()
+  const navigation = useNavigation()
 
   const dispatch = useDispatch()
   const handleSubmit = useCallback(
     (values: CreatePlaylistValues) => {
       dispatch(createPlaylist(values, CreatePlaylistSource.FAVORITES_PAGE))
       toast({ content: messages.playlistCreatedToast })
+      navigation.goBack()
```

Comment:
oh i might have removed this? thanks

Reply:
no worries! lets leave this in for another week, but chatted with sebastian about a better way long term